### PR TITLE
Create Organization Readme.md

### PR DESCRIPTION
**Pull request by pransh15**
## About the changes

Added a GitHub organization Readme for Unleash with all the relevant links for devs to be able to start the right way.

Should look something like this:

image

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+</p>
+
+![Unleash_GitHub_Mocks (2)](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
+
+
+<!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->
+
+**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases, and, in the beginning, we were just a few contributors. Together we built a foundation for the amazing community that you see today. 🫂
+
+We love that our community is able to work with our product with all the freedom you could ever want. When you can just test, experiment, and release confidently, you'll end up with a well-rounded product with a ton of awesome interpretations. We're proud of you guys. 🙏
```
Comment (sighphyre) [inline profile/README.md]: I'm not quite sure what interpretations means here?

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+[Docker Hub](https://hub.docker.com/r/unleashorg/unleash-server) - Unleash containerized, just for you. 🎛️ </br>
+[Unleash Docs](https://docs.getunleash.io) - Dive deeper into the docs. 📚 </br>
+[How-to Guides](https://docs.getunleash.io/how-to) - All the how-to's answered for you. 🙋 </br>
+[Open Live Demo](https://www.getunleash.io/interactive-demo) - Unleash Official demo application using Unleash + Next.js.</br>
+[Feature Flagging - Why and How to Add to Your Software](https://youtu.be/-yHZ9uLVSp4) - Need something visual to get started? Here's something to watch. 📹 </br>
+
+## ✨ Connect with Us</br>
+Get involved with a vibrant community of users and contributors. Check out what others have built with Unleash, start learning by going through Unleash How-to Guides or get involved as a contributor.
+
+[Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo 🌟 </br>
```
Comment (sighphyre) [inline profile/README.md]:
```suggestion
[Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo. 🌟 </br>
```

**Review by sighphyre — COMMENTED** [review]
I think this is cool!

File: profile/README.md (hunk 1 above)
Comment (pransh15) [reply profile/README.md]: Interpretations here would mean, to each their own. Calling Unleash as a versatile product to be used anywhere in their stack.

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+![Unleash_GitHub_Banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)
+
+<p align="center">
+<b>The Open Source Feature Flagging Service</b>
+</p>
+
+![Unleash_GitHub_Mocks (2)](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
+
+
+<!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->
```
Comment (thomasheartman) [inline profile/README.md]: Any reason this is commented out? Could we just remove it?

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+![Unleash_GitHub_Banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)
```
Comment (thomasheartman) [inline profile/README.md]:
```suggestion
![Unleash GitHub banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)
```

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+<p align="center">
+<b>The Open Source Feature Flagging Service</b>
+</p>
+
+![Unleash_GitHub_Mocks (2)](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
```
Comment (thomasheartman) [inline profile/README.md]: Hmm, I'm not sure how best to describe this image. Something like this, perhaps?

```suggestion
![Unleash marketing image showing "Flexible segmentation", "Feature toggles", "Projects" and that Unleash integrates with many programming languages.](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
```

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+
+<!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->
+
+**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases, and, in the beginning, we were just a few contributors. Together we built a foundation for the amazing community that you see today. 🫂
```
Comment (thomasheartman) [inline profile/README.md]: Ever since being told by my (Scottish) English teacher in high school that "in the beginning" is very specific to the book of Genesis and that you should always use "at the beginning" if you're not invoking religious imagery, I've tried to avoid it. I did some research now, and from what I can tell, it definitely still carries with it some of that meaning, but it's also not a 100%, so I'm not gonna be completely opposed 🤷🏼 

Also: "We wanted to fill an essential space in the development stack" Did we? I thought Ivar was just solving a need he had himself?

Regardless, I think we can re-jig these sentences some. How about this?

```suggestion
**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases. At first, we were just a few contributors, but together we built a foundation for the amazing community that you see today. 🫂
```

File: profile/README.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+> [📈 See our roadmap →](https://github.com/orgs/Unleash/projects/5)
+
+## 🚀 Learn about Unleash
+Create with Freedom, Release With Confidence.
+
+[Main Open Source Repo](https://github.com/Unleash/unleash) - Unleash's Open Source codebase. 🖥️ </br>
+[Docker Hub](https://hub.docker.com/r/unleashorg/unleash-server) - Unleash containerized, just for you. 🎛️ </br>
+[Unleash Docs](https://docs.getunleash.io) - Dive deeper into the docs. 📚 </br>
+[How-to Guides](https://docs.getunleash.io/how-to) - All the how-to's answered for you. 🙋 </br>
+[Open Live Demo](https://www.getunleash.io/interactive-demo) - Unleash Official demo application using Unleash + Next.js.</br>
```
Comment (thomasheartman) [inline profile/README.md]: Wait, does this use Next.js? I wasn't aware of that 😅

File: profile/README.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+## ✨ Connect with Us</br>
+Get involved with a vibrant community of users and contributors. Check out what others have built with Unleash, start learning by going through Unleash How-to Guides or get involved as a contributor.
+
+[Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo. 🌟 </br>
+[GitHub Discussions](https://github.com/Unleash/unleash/discussions) - Place to discuss, ask questions and find answers, show your Unleash project and get feedback. 🗣️ </br>
+[Slack](https://unleash-community.slack.com/) -  Come chat with the Unleash community including the core team. 💬 </br>
+[Community Homepage](https://www.getunleash.io/unleash-community) - Check out the ways you can be a valuable member of the community. 🛰️ </br>
+<!-- [Events at Unleash]() - Learn more about new Events for the Unleash Community. 🎪 </br> -->
+Contributing Guidelines coming soon. ⚠️ </br>
```
Comment (thomasheartman) [inline profile/README.md]: Any reason you're using regular sentences instead of a list here? It looks like a list to me, so I think the right way to do this would be to use a list. It would also be helpful for people using assistive tech to read this, so that it's correctly called out as a list 💁🏼 

I'd usually also remove comments, but this seems more like a placeholder for now?

```suggestion
- [Main Open Source Repo](https://github.com/Unleash/unleash) - Unleash's Open Source codebase. 🖥️
- [Docker Hub](https://hub.docker.com/r/unleashorg/unleash-server) - Unleash containerized, just for you. 🎛️ 
- [Unleash Docs](https://docs.getunleash.io) - Dive deeper into the docs. 📚
- [How-to Guides](https://docs.getunleash.io/how-to) - All the how-to's answered for you. 🙋
- [Open Live Demo](https://www.getunleash.io/interactive-demo) - Unleash Official demo application using Unleash + Next.js
- [Feature Flagging - Why and How to Add to Your Software](https://youtu.be/-yHZ9uLVSp4) - Need something visual to get started? Here's something to watch. 📹

## ✨ Connect with Us

Get involved with a vibrant community of users and contributors. Check out what others have built with Unleash, start learning by going through Unleash How-to Guides or get involved as a contributor.

- [Leave us a Star](https://github.com/Unleash/unleash) on the Unleash Repo. 🌟
- [GitHub Discussions](https://github.com/Unleash/unleash/discussions) - Place to discuss, ask questions and find answers, show your Unleash project and get feedback. 🗣️
- [Slack](https://unleash-community.slack.com/) -  Come chat with the Unleash community including the core team. 💬 
- [Community Homepage](https://www.getunleash.io/unleash-community) - Check out the ways you can be a valuable member of the community. 🛰️ 
<!-- - [Events at Unleash]() - Learn more about new Events for the Unleash Community. 🎪 -->
- Contributing Guidelines coming soon. ⚠️
```

File: profile/README.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+We can't wait to join you on this journey 🫶
```
Comment (thomasheartman) [inline profile/README.md]: Are we joining them or are they joining us? I feel like it's the other way around, but I'm open to hearing opposing views on this 😄 

```suggestion
We can't wait for you to join us on this journey 🫶
```

File: profile/README.md (hunk 1 above)
Comment (thomasheartman) [inline profile/README.md]: A few minor suggestions on this paragraph:

I think we should avoid gender-specific terminology where possible. While "guys" is often used in a gender-neutral way, it's also subject to heavy debate. Consider other options where applicable or just drop it completely.

I'm also with Simon on the use of "interpretations" here: it's a little unclear. Could we 

I'm also confused about who "you" are in this sentence. It both appears to refer to the general, unspecified "you" (which even sounds a bit like Unleash) **and** to the community.

I'd suggest rewording this a bit. I've take a stab at it here, but feel free to discard this if you have any better ideas ☺️ 

```suggestion
We love that you — our community — have the freedom to work with our product in any way you want. Together we can test, experiment, and create a well-rounded product that covers a ton of different use cases. We're super proud of you 🙏
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Super work and thanks so much for doing this 🙌🏼  I've got a few minor change requests plus one paragraph I think we might want to reword a bit, but otherwise it's great 👏🏼

File: profile/README.md (hunk 9 above)
Comment (sighphyre) [reply profile/README.md]: Same, I think Thomas' suggestion here reads better but also open to opposing views

File: profile/README.md (hunk 1 above)
Comment (sighphyre) [reply profile/README.md]: I like this! I spent too long thinking about wording I liked here and came up with nothing, really nice suggestion, sir

File: profile/README.md (hunk 3 above)
Comment (pransh15) [reply profile/README.md]: We can, I was testing it out so I probably missed it during the final PR.

File: profile/README.md (hunk 7 above)
Comment (pransh15) [reply profile/README.md]: I believe it does, it's a small detail we can update it later.

File: profile/README.md (hunk 8 above)
Comment (pransh15) [reply profile/README.md]: I don't have a strong preference here, to be honest. Let's move forward with lists.

The comment is a placeholder, yes!

File: profile/README.md (hunk 1 above)
Comment (pransh15) [reply profile/README.md]: This is definitely more readable, and puts everything in context. I don't think I can do better than this.

File: profile/README.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
+![Unleash GitHub banner](https://user-images.githubusercontent.com/18158596/196736157-b20a5fbf-74ac-47cd-a991-fe6e2d2bc960.png)
+
+<p align="center">
+<b>The Open Source Feature Flagging Service</b>
+</p>
+
+![Unleash marketing image showing "Flexible segmentation", "Feature toggles", "Projects" and that Unleash integrates with many programming languages.](https://user-images.githubusercontent.com/18158596/196740001-569c6951-e989-4f7e-b31b-def5c40fa2aa.png)
+
+
+<!-- <img width="600" alt="Unleash Logo" src="https://user-images.githubusercontent.com/18158596/196730085-d168ff9e-8b1a-4790-8219-882336df6479.png"> -->
```
Comment (thomasheartman) [inline profile/README.md]: If we can remove this, then let's remove it before merging 😄 

```suggestion
```

File: profile/README.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+
+**Unleash started as a side project.** We wanted to fill an essential space in the development stack and control releases. At first, we were just a few contributors, but together we built a foundation for the amazing community that you see today. 🫂
+
+We love that you — our community — have the freedom to work with our product in any way you want. Together we can test, experiment, and create a well-rounded product that covers a ton of different use cases. We're super proud of you 🙏
+
+That's enough about us. Have fun. Be kind. Let's **Unleash** great software together. 💪
+
+> [📈 See our roadmap →](https://github.com/orgs/Unleash/projects/5)
```
Comment (thomasheartman) [inline profile/README.md]: Oh, wait, we actually created a new roadmap, so we should update that link too 🤓 Also, I suggest changing "see" to "check out" or similar for non-sighted users. 

```suggestion
> [📈 Check out our roadmap →](https://github.com/orgs/Unleash/projects/10)
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Great work! Just two more bits and it's ready to go from what I can tell 🔥

**Review by pransh15 — COMMENTED** [review]
LGTM! 🚀

**Review by thomasheartman — APPROVED** [review]
Sweet! Let's get it in 🚂
